ansible-ipi-install/roles/node-prep/tasks/09_create-install-config.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Check if Pull Secret is set
```

Comment:
shouldn't install-config generation be a "per cluster" operation that takes place as part of the installer steps rather than during node-prep. It's feasible that one provisioning host can be used for multiple clusters.

Reply:
Good question. I was always under the impression it was a 1 cluster per prov host but seems like that isn't the only use case.

Reply:
i think it's pretty likely we'll encounter a situation where we need one provisioning host to be capable of multiple cluster provisioning... It's definitely an outlier use case though.

Reply:
I think multiple cluster is an interesting use case, but for the purposes of this playbook, I'm going to designate that a feature enhancement.